readme.md

```diff
@@ -92,7 +105,11 @@ Using Ruby, instantiate an array called `fruits` that contains `apple`, `banana`
 Then, use an enumerator to print to the console the sentence "I'd like to eat a [fruit]" once for each fruit.
 
 ```rb
-# Your answer...
+# fruits = ['apple', 'banana', 'orange']
+
+fruits.each do |fruit|
+    puts I'd like to eat a |fruit|
+end
 
 ```
 
```

Comment:
You do not use the `||` to reference the iteration variable inside of the enumerator. You can just use the variable name by itself. So it would look like this...

```ruby
fruits.each do |fruit|
  puts "I'd like to eat a #{fruit}"
end
```

Also make sure that you're `puts`ing a string. This code would error out.